.kitchen.yml

```diff
@@ -23,62 +23,114 @@ platforms:
     driver:
       vm_hostname: origin-centos-72
 
+openshift3-shared-attributes: &SHARED
+  # uncomment to use experimental CentOS PaaS repository for integration tests
+  #yum_repositories:
+  #  - name: "centos-openshift-origin"
+  #    baseurl: "http://buildlogs.centos.org/centos/7/paas/x86_64/openshift-origin/"
+  #    gpgcheck: false
+  # we override these because 10.0.2.15 is whitelisted in $no_proxy
+  openshift_common_public_hostname: 10.0.2.15
+  openshift_master_router_subdomain: cloudapps.10.0.2.15.nip.io
+  openshift_master_metrics_public_url: metrics.10.0.2.15.nip.io
+  openshift_common_default_nodeSelector: region=infra
+  docker_log_driver: journald
+  persistent_storage:
+  - name: testpv
+    capacity: 100Mi
+    access_modes: ReadOnlyMany
+    path: /srv/testpv
+    server: 10.0.2.15
+    claim:
+      namespace: default
+  - name: registry-storage
+    capacity: 100Mi
+    access_modes: ReadWriteMany
+    path: /srv/registry-storage
+    server: 10.0.2.15
+    claim:
+      namespace: default
+  registry_persistent_volume: registry-storage
+  openshift_hosted_metrics_parameters:
```

Comment:
Here are all hosted registry parameters, including HAWKULAR_METRICS_HOSTNAME. I only override the IMAGE_VERSION in the version-specific openshift3-base roles.